phpstan.neon.dist

```diff
@@ -120,8 +130,12 @@ parameters:
             paths:
                 - lib/Doctrine/DBAL/Driver/OCI8/OCI8Statement.php
 
-        # https://github.com/phpstan/phpstan-src/pull/700
         -
-            message: '~^Parameter #2 \$count of function array_fill expects int<0, max>, int given\.$~'
+            message: '~Method Doctrine\\DBAL\\Driver\\Mysqli\\MysqliStatement::rowCount\(\) should return int but returns int\|string\.~'
             paths:
                 - lib/Doctrine/DBAL/Driver/Mysqli/MysqliStatement.php
+
+        -
+            message: '~Dead catch - PDOException is never thrown in the try block\.~'
```

Comment:
Where exactly is this? Should we document the missing exception instead?

Reply:
In `SQLServerSchemaManager::listTableIndexes()`. Looking at the code, PHPStan might be right, actually. I've pushed a commit that removed the catch. Let's see if the tests still pass. 🧐